app/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.8
 ENV PYTHONUNBUFFERED 1
 
 RUN apt-get update
-RUN apt-get install ffmpeg libsm6 libxext6 libhealpix-cxx-dev -y
+RUN apt-get install ffmpeg libsm6 libxext6 libhealpix-cxx-dev gfortran -y
```

Comment:
It appears that I can run the slim and full stack in your branch without this addition to the docker file. By using BLAST_IMAGE=blast_latest, can you confirm you can do this also? If you can please remove gfortran from the docker file :)